packaging/Dockerfile.fedora-nmstate-dev

```diff
@@ -2,7 +2,7 @@
 # Fedora official repository
 # (https://hub.docker.com/r/fedora/systemd-systemd/).
 # It enables systemd to be operational.
-FROM docker.io/library/fedora:31
```

Comment:
This does not seem to belong here

Reply:
Whoops, I am opening a new PR for this. Thanks!